### PR TITLE
TripCard component updates

### PR DESCRIPTION
**Pull request by mcellier**
- Added shadow
- Added badge option
- Added status information option 
- Added multiple passengers display

Kapture 2019-06-06 at 16 56 40

### CODE REVIEW

**Comment by mcellier** [comment]
Note: i have to fix some typescript issues again

**Comment by pith** [comment]
The margin top for the status seems too big

File: src/tripCard/index.tsx [hunk 1]
```diff
@@ -82,19 +99,56 @@ const TripCard = ({
...
             </Text>
           </div>
           <div className="kirk-tripCard-bottom">
-            <div className="kirk-tripCard-driver">
-              <div className="kirk-tripCard-avatar">
-                <Avatar image={driver.avatarUrl} alt="" />
+            {driver && (
+              <div className="kirk-tripCard-driver">
+                <div className="kirk-tripCard-avatar">
+                  <Avatar image={driver.avatarUrl} alt="" />
```
Comment (pith) [inline src/tripCard/index.tsx]: `alt=""` seems unnecessary

File: src/tripCard/index.tsx [hunk 2]
```diff
@@ -82,19 +99,56 @@ const TripCard = ({
...
+                </div>
+                <Text display={TextDisplayType.TITLE}>{driver.firstName}</Text>
               </div>
-              <Text display={TextDisplayType.TITLE}>{driver.firstName}</Text>
-            </div>
+            )}
+            {passengers && (
+              <ul className="kirk-tripCard-passengers">
+                {passengers.reverse().map(passenger => (
```
Comment (pith) [inline src/tripCard/index.tsx]: Nice 👍

File: src/tripCard/specifications/index.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Primary Button
```
Comment (pith) [inline src/tripCard/specifications/index.md]: 🤔 wrong copy/paste ?

File: src/tripCard/specifications/index.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+```js
+import { TripCard } from '@blablacar/ui-library'
```
Comment (pith) [inline src/tripCard/specifications/index.md]: Add also a sample of usage (unless it's already the case with the story)

File: src/tripCard/style.ts [hunk 5]
```diff
@@ -3,9 +3,17 @@ import { color, radius, space, componentSizes } from '_utils/branding'
 
 export default css`
   li {
+    list-style-type: none;
```
Comment (pith) [inline src/tripCard/style.ts]: Doesn't seem the good place for that

File: src/tripCard/style.ts [hunk 6]
```diff
@@ -32,19 +60,33 @@ export default css`
...
-  .kirk-tripCard-driver {
+  .kirk-tripCard-driver,
+  .kirk-tripCard-passengers {
     display: flex;
     align-items: center;
     flex: 1;
     margin-right: ${space.l};
   }
 
+  /* We reverse the passengers array to display the 1st one on top without touching z-indexes */
```
Comment (pith) [inline src/tripCard/style.ts]: Thanks for the comment ! And nice trick 😉

File: src/tripCard/specifications/index.md (hunk 4 above)
Comment (mcellier) [reply src/tripCard/specifications/index.md]: It is

File: src/tripCard/index.unit.tsx [hunk 7]
```diff
@@ -38,32 +40,76 @@ const mockedProps = {
...
+        avatarUrl: '//placehold.it/500x500',
+        firstName: 'Matthew',
+      },
+    ]
+
+    const component = shallow(<TripCard {...mockedProps} driver={null} passengers={passengers} />)
+    expect(component.find(Avatar)).toHaveLength(3)
+  })
+
+  it('Should render status informations', () => {
```
Comment (Tuxosaurus) [inline src/tripCard/index.unit.tsx]: `information`

File: src/tripCard/specifications/index.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+## Playground
+
+<!-- STORY -->
+
+## Specifications
+
+> Card
+
+This component serves as a container for other components. It entire area becomes clickable.
```
Comment (Tuxosaurus) [inline src/tripCard/specifications/index.md]: `It's entire area`

**Pull request merged**